templates/tempest.j2

```diff
@@ -1,6 +1,6 @@
 [DEFAULT]
-swift_operator_role = member
-swift_reseller_admin_role = admin
+swift_operator_role = {{ rally_tempest_swift_operator_role }}
```

Comment:
These kind of  change brakes backwards compatibility. I'm not 100% sure of the syntax but I think you should be able to do something like this:
swift_operator_role = {{ rally_tempest_swift_operator_role |default('member') }}

Reply:
Added vars in defaults/main.yml